runbooks/KubevirtHyperconvergedClusterOperatorCRModification.md

```diff
@@ -0,0 +1,30 @@
+# KubevirtHyperconvergedClusterOperatorCRModification
+
+## Meaning
+
+Hyperconverged Cluster Operator configures clusters in an opinionated way, and it overwrites its operands when there is an unexpected change on them. Users are expected to not modify the operands directly. The `HyperConverged` object is the source of truth for the configuration.
+
+This alert fires when an operand of Hyperconverged Cluster Operator is changed by someone else.
+
+
+## Impact
+
+When the operands are changed by someone else constantly, it may lead to oscillation in the cluster configuration and instabilities.
+
+
+## Diagnosis
+
+Check the alert details. `component_name` contains the type and name of operand that is overwritten.
```

Comment:
`component_name` contains the type and name of the operand that is overwritten.

Reply:
done